### PR TITLE
Add feature for adding keys by environment variables

### LINKED ISSUE #104: Provide credentials thru environment variable?
It is often useful in CI systems etc to provide credentials via an environment variable. Is this possible in dbxcli?

PS: Holy crap @diwakergupta its been a long time and no see! ✋

### PR DESCRIPTION
**Pull request by vaithak**
Fixes #104 .

### CODE REVIEW

**Comment by vaithak** [comment]
@diwakergupta Please review my PR.

File: cmd/root.go [hunk 1]
```diff
@@ -39,13 +39,21 @@ const (
...
+}
+
 var (
-	personalAppKey      = "mvhz183vwqibe7q"
-	personalAppSecret   = "q0kquhzgetjwcz1"
-	teamAccessAppKey    = "zud1va492pnehkc"
-	teamAccessAppSecret = "p3ginm1gy0kmj54"
-	teamManageAppKey    = "xxe04eai4wmlitv"
-	teamManageAppSecret = "t8ms714yun7nu5s"
+	personalAppKey      = getEnv("DROPBOX_PERSONAL_APP_KEY", "mvhz183vwqibe7q")
```
[the remark this answers is not in the document]
Comment (vaithak) [reply cmd/root.go]: Will changing the name of env variables in root.go to that of build.sh work ?

**Comment by vaithak** [comment]
@diwakergupta I have made the requested changes, now the keys can be accessed through env variables without build failure in official release. Please review my PR.

File: cmd/root.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (vaithak) [reply cmd/root.go]: Ok! I have done the requested changes, please review them.

**Comment by clarkewd** [comment]
Thanks for adding this feature! I've looked over the changes and also the code in `root.go`, and read the comments here and on https://github.com/dropbox/dbxcli/issues/104

I'm still a little confused though. Is it possible to continue to use the default app keys (i.e. not create my own dropbox app) and then use either the web authentication response or the data from the `~/.config/dbxcli/auth.json` file  set as an environmental variable instead of using the config file?

I've tried it with both `DROPBOX_PERSONAL_APP_KEY` and `DROPBOX_PERSONAL_APP_SECRET` but this doesn't seem like the right place for it...

**Pull request merged**